src/Integrations/NetworkTestResult.ts

```diff
@@ -0,0 +1,4 @@
+export interface NetworkTestResult {
+  output : string,
+  repeaterId: string
+}
```

Comment:
Don't disable `husky` (i.e git hooks), it prevents you to avoid such mistakes.

Reply:
i didnt disable smthg. how i can enable it?

Reply:
It is enabled by default. Try to remove `.git/hooks` and re-install deps using `npm cli`